nautobot/dcim/tests/test_views.py

```diff
@@ -1290,7 +1282,7 @@ def setUpTestData(cls):
             "comments": "A new device",
             "tags": [t.pk for t in Tag.objects.get_for_model(Device)],
             "local_config_context_data": None,
-            "cf_crash-counter": -1,
+            "cf_crash_counter": -1,
             "cr_router-id": None,
```

Comment:
Why we need similar validation and enforcement for relationship ~slugs~ keys, exhibit 1 of N.